mobile-app/app/screens/AppNavigator/screens/Loans/screens/EditCollateralScreen.tsx

```diff
@@ -113,33 +112,16 @@ export function EditCollateralScreen ({
     dismissModal()
     const isConversionRequired = item.token.id === '0' ? new BigNumber(item.amount).gt(DFIToken.amount) : false
```

Comment:
withdrawal case always no conversion require

Reply:
Yep there's no conversion for withdrawal. it's just add/remove are using the same component for collaterals

Reply:
For the check, it only happened twice, (EditCollateralScreen and on the bottom sheet token, when you select a token from the list)